src/app/clinic-dashboard/clinic-dashboard.guard.ts

```diff
@@ -27,8 +31,10 @@ export class ClinicDashboardGuard implements CanActivate, CanDeactivate<ClinicDa
           },
           routes: []
         });
+      } else if (userLocation && userLocation.uuid) {
+        this.router.navigate(['/clinic-dashboard', userLocation.uuid, 'daily-schedule']);
       } else {
-        this.router.navigate(['/patient-dashboard/patient-search']);
+        this.router.navigate(['/user-default-properties']);
```

Comment:
user should be allowed to select the location just as it is, and given freedom to switch locations easily. Allow navigation into clinic dashboard without requiring the user to select user default property.